src/extension.ts

```diff
@@ -77,54 +76,49 @@ export async function activate(context: vscode.ExtensionContext) {
         registerCommand({
             command: 'aws.login',
             callback: async () => await ext.awsContextCommands.onCommandLogin(),
-            telemetryName: {
-                namespace: TelemetryNamespace.Aws,
-                name: 'credentialslogin'
-            }
+            telemetryName: 'aws_credentialslogin'
         })
 
         registerCommand({
             command: 'aws.credential.profile.create',
             callback: async () => await ext.awsContextCommands.onCommandCreateCredentialsProfile(),
-            telemetryName: {
-                namespace: TelemetryNamespace.Aws,
-                name: 'credentialscreate'
-            }
+            telemetryName: 'aws_credentialscreate'
         })
 
         registerCommand({
             command: 'aws.logout',
             callback: async () => await ext.awsContextCommands.onCommandLogout(),
-            telemetryName: {
-                namespace: TelemetryNamespace.Aws,
-                name: 'credentialslogout'
-            }
+            telemetryName: 'aws_credentialslogout'
         })
 
         // register URLs in extension menu
         registerCommand({
             command: 'aws.help',
             callback: async () => {
                 vscode.env.openExternal(vscode.Uri.parse(documentationUrl))
-            }
+            },
+            telemetryName: 'Command_aws.help'
```

Comment:
Do we not have a similar metric for Jetbrains? If so, then do `aws_help` (or match what they have)

Reply:
This PR contains no metric name changes, those will be in a later PR